sw/device/silicon_creator/lib/sigverify/sphincsplus/address_unittest.cc

```diff
@@ -0,0 +1,259 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/silicon_creator/lib/sigverify/sphincsplus/address.h"
+
+#include <array>
+
+#include "gtest/gtest.h"
+#include "sw/device/silicon_creator/testing/rom_test.h"
+
+namespace spx_addr_unittest {
+namespace {
+
+class SpxAddrTest : public rom_test::RomTest {
+ protected:
+  /**
+   * Fetch a specific byte from the internal address buffer.
+   *
+   * @param addr Hypertree address.
+   * @param offset Index of byte to get.
+   * @return Value of the byte within the internal buffer.
+   */
+  uint8_t byte_get(const spx_addr_t *addr, size_t offset) {
```

Comment:
nit: c++ function names should be pascal case to comply with the style guide:
```suggestion
  uint8_t ByteGet(const spx_addr_t *addr, size_t offset) {
```

Applies to other functions as well.